app/views/Dashboard/Overview/MapView/index.tsx

```diff
@@ -22,7 +22,7 @@ import MapLabel from '#components/MapLabel';
 import {
     normalFormatter,
 } from '#utils/common';
-// import { RegionBounds } from '#utils/regionBounds';
+import { RegionBounds } from '#utils/regionBounds';
```

Comment:
```suggestion
import { regionBounds } from '#utils/regionBounds';
```
Please change this in the definition part as well.